### PR TITLE
Philippika

### PR DESCRIPTION
**Pull request by NMonteix**
Citation style for the Philippika collection (Harrassowitz Verlag)

### CODE REVIEW

File: philippika.csl [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+    </author>
+    <category citation-format="note"/>
+    <category field="history"/>
+    <category field="humanities"/>
+    <issn>1613-5628</issn>
+    <summary>Citation style for the Philippika collection (Harrassowitz Verlag). Language normally set as German. Using Zotero, to set English, use the Quick Locale Switcher (see https://www.zotero.org/support/supported_languages)</summary>
+    <updated>2015-03-06T23:37:43+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="de"/>
```
Comment (adam3smith) [inline philippika.csl]: delete--this doesn't do anything.

File: philippika.csl [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+  <macro name="author">
+    <names variable="author" delimiter=", ">
+      <name delimiter-precedes-last="never" name-as-sort-order="all" and="symbol" sort-separator=", " initialize-with="." delimiter=", " et-al-min="3" et-al-use-first="1"/>
+      <et-al/>
```
Comment (adam3smith) [inline philippika.csl]: delete.

File: philippika.csl [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+      <substitute>
+        <text macro="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor" delimiter=", ">
+      <name delimiter-precedes-last="never" name-as-sort-order="all" and="symbol" sort-separator=", " initialize-with="." delimiter=", " et-al-min="3" et-al-use-first="1"/>
+      <et-al/>
```
Comment (adam3smith) [inline philippika.csl]: delete

File: philippika.csl [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+      <label form="short" prefix=" (" text-case="lowercase" suffix=")"/>
+      <substitute>
+        <names variable="container-author" delimiter=", ">
```
Comment (adam3smith) [inline philippika.csl]: ```<names variable="container-author"/>``` is all you need here. Delete the rest.

File: philippika.csl [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+      <number variable="volume" form="numeric"/>
+    </group>
+  </macro>
+  <macro name="pages">
+    <text variable="page"/>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name delimiter-precedes-last="never" name-as-sort-order="all" and="symbol" form="short" delimiter=", " et-al-min="3" et-al-use-first="1"/>
+      <et-al font-variant="normal"/>
```
Comment (adam3smith) [inline philippika.csl]: delete.

File: philippika.csl [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+  <macro name="publisher-date">
+    <group>
+      <text macro="publisher" prefix=", " suffix=" "/>
+      <text macro="edition"/>
+      <text macro="year-date"/>
+    </group>
+  </macro>
+  <macro name="access">
+    <group>
+      <text term="accessed" suffix=" "/>
```
Comment (adam3smith) [inline philippika.csl]: in German that gives you e.g. 7 März 2015 -- that seems wrong. You probably want to use a localized date here, like ```<date variable="accessed" form="numeric"/>``` and then no date parts.

File: philippika.csl [hunk 7]
```diff
@@ -0,0 +1,257 @@
...
+          <if locator="page" match="none">
+            <label variable="locator" form="short" suffix=" "/>
+          </if>
+        </choose>
+        <text variable="locator"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="title">
+    <choose>
```
Comment (adam3smith) [inline philippika.csl]: none of this does anything. Simplify the whole macro to 
```<text variable="title" text-case="title"/>```

**Comment by adam3smith** [comment]
I'd ask you to remove all the et-al-min and et-al-use-first attributes from the names and just put them on bibliography and citation.

Also--the way you have this now, the language will just default to the user's default. Does that really make sense here? Wouldn't we just want to set a default locale to de? Almost all of Philippika seems to be in German

**Comment by NMonteix** [comment]
Done every corrections. As for the date, I don't have a German example with me from this collection, but I guess it should work as corrected.
Given I had to change the German form of et-al to a Latin one, I inserted back the locale xml:lang=de.
If I had not done this, how could I set the default locale to de?
If anything appears fine to you, maybe wait until tomorrow to merge it, I have to change the calling for encyclopedia articles (tomorrow...)

**Comment by adam3smith** [comment]
looks good to me. You changed the non-breaking spaces to regular spaces on purpose?

**Comment by NMonteix** [comment]
Maybe not... I'll check it preparing the encyclopaedia articles...

**Comment by NMonteix** [comment]
l. 212 sq., I had to use this
              <choose>
                <if variable="title-short" match="any">
                  <text variable="title-short" font-style="italic"/>
                </if>
                <else>
                  <text variable="container-title" font-style="italic"/>
                </else>
              </choose>
to obtain this
Wagner 1894 = Wagner, R., s.v. Antilochus, RE, 2 (1894), 2429–2431.
Because using this
              <text variable="title-short" font-style="italic"/>
              <substitute>
                <text variable="container-title" font-style="italic"/>
              </substitute> 
I obtained this
Wagner 1894 = Wagner, R., s.v. Antilochus, RE, Realencyclopädie der classischen Altertumswissenschaft, 2 (1894), 2429–2431.

I guess with a container-title-short, it would be easier
Checked for non-breaking space. Now they fit well.

**Comment by NMonteix** [comment]
How can I make lines of code being visible in discussions?

**Comment by NMonteix** [comment]
There was a sub-verbo problem in the previous version. Strangely enough, it worked perfectly in the Zotero style editor.

**Comment by adam3smith** [comment]
yeah, you can use substitute only as part of ```names```. 
(to cite code wrap it in three ` on each side. Otherwise it vanishes as it did in your post above. 
This looks good to me now. Not quite sure I understand your last post--is this ready to be merged?

**Comment by NMonteix** [comment]
Now, yes, thanks
In the previous version, I had a problem in giving peculiar form of sub-verbo not in hard: it worked with Zotero, not with Travis. Now it works, hard coded...

**Comment by adam3smith** [comment]
OK, great, thanks!

**Comment by NMonteix** [comment]
To you

**Pull request merged**